src/utilities/fastify/server.ts

```diff
@@ -43,13 +43,17 @@ export async function makeFastify(appPlugin: FastifyPluginAsync, customLogger?:
 
   const internalServerError = 'Internal server error';
   server.setErrorHandler(async (error, _request, reply) => {
-    if (error.statusCode !== undefined && error.statusCode < HTTP_STATUS_CODES.INTERNAL_ERROR) {
+    if (
+      error.statusCode !== undefined &&
+      error.statusCode < HTTP_STATUS_CODES.INTERNAL_SERVER_ERROR
+    ) {
+      logger.info(error, 'Client error');
```

Comment:
I just remembered that Pino requires errors to be called `err`:

```suggestion
      logger.info({err: error}, 'Client error');
```